src/Raven.Server/Documents/Sharding/ShardedContext.cs

```diff
@@ -154,5 +164,56 @@ public bool HasClientConfigurationChanged(long clientConfigurationEtag)
             var actual = Hashing.Combine(lastClientConfigurationIndex, _lastClientConfigurationIndex);
             return actual > clientConfigurationEtag;
         }
+
+        public string GetShardedDatabaseName(int index = 0)
```

Comment:
move this to some helper class, we do have ClientShardHelper and ShardHelper

Reply:
removed